OrbitGl/TracepointTrack.cpp

```diff
@@ -76,7 +76,11 @@ void TracepointTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
           float radius = track_height / 4;
           Vec2 pos(time_graph_->GetWorldFromTick(time), pos_[1]);
           if (thread_id_ == TracepointEventBuffer::kAllTracepointsFakeTid) {
-            batcher->AddVerticalLine(pos, -track_height, z, kWhiteTransparent);
+            if (tracepoint.is_in_target_process() == false) {
+              batcher->AddVerticalLine(pos, -track_height, z, kWhiteTransparent);
+            } else {
+              batcher->AddVerticalLine(pos, -track_height, z, kGreenSelection);
```

Comment:
I'm not too sure about this green color, especially since it's already used for the selection. Not too sure what to suggest though. I would stay along the lines of white for one and grey for the other, or white for one and white but even more transparent for the other.

Reply:
Also, for compactness, I would suggest:

Color color = tracepoint.is_in_target_process() ? kGreenSelection : kWhiteTransparent;
batcher->AddVerticalLine(pos, -track_height, z, color);

Reply:
Thank you!